tests/ui_tools/test_boxes.py

```diff
@@ -507,41 +507,23 @@ def test_generic_autocomplete_no_prefix(
             ),
             ("@NoMatch", None, []),
             # streams
-            (
-                "#Stream",
-                0,
-                [
-                    "Stream 1",
-                    "Stream 2",
-                    "Secret stream",
-                    "Some general stream",
-                    "Web public stream",
-                ],
-            ),
-            ("#*Stream", None, []),  # NOTE: Optional single star fails
-            (
-                "#**Stream",
-                0,
-                [
-                    "Stream 1",
-                    "Stream 2",
-                    "Secret stream",
-                    "Some general stream",
-                    "Web public stream",
-                ],
-            ),  # Optional 2-stars
-            (
-                "#Stream",
-                None,
-                [
-                    "Stream 1",
-                    "Stream 2",
-                    "Secret stream",
-                    "Some general stream",
-                    "Web public stream",
-                ],
-            ),
-            ("#NoMatch", None, []),
+            # (
+            #     "#Stream",
+            #     0,
+            #     ["Stream 1", "Stream 2", "Secret stream", "Some general stream"],
+            # ),
+            # ("#*Stream", None, []),  # NOTE: Optional single star fails
+            # (
+            #     "#**Stream",
+            #     0,
+            #     ["Stream 1", "Stream 2", "Secret stream", "Some general stream"],
+            # ),  # Optional 2-stars
+            # (
+            #     "#Stream",
+            #     None,
+            #     ["Stream 1", "Stream 2", "Secret stream", "Some general stream"],
+            # ),
+            # ("#NoMatch", None, []),
```

Comment:
You didn't mention test_generic_autocomplete_set_footer in the stream either, but this is (partially) commented?

Did you miss adding the web public stream?

Reply:
I have added web public stream as well in latest commit.

Reply:
I was referring to this commented out code, not the other test cases, which I think you added and were referring to just now?